source/api/presentation/3.0/index.md

```diff
@@ -316,50 +319,30 @@ The value _MUST_ be an [`xsd:dateTime` literal][xsd-datetime]. The value _MUST_
 { "navDate": "2010-01-01T00:00:00Z" }
 ```
 
-##### rights
-
-A link to an external resource that describes the license or rights statement under which this resource may be used. The rationale for the value being a URI and not a human readable text is that typically there is one license for many resources, and the text is too long to be displayed to the user at the same time as the object. If displaying rights information directly to the user is a requirement, then it is _RECOMMENDED_ to include the information using the `requiredStatement` property.
-
-The value _MUST_ be an array of JSON objects, each of which _MUST_ have an `id` and _SHOULD_ have at least one of `type` and `format`.
-
- * Any resource type _MAY_ have the `rights` property with at least one item.<br/>
-   Clients _MUST_ render `rights` on every resource type.
-
-``` json-doc
-{
-  "rights": [
-    {
-      "id": "https://example.org/rights/copyright.html",
-      "type": "Text",
-      "format": "text/html"
-    }
-  ]
-}
-```
 
-###  3.3. Technical Properties
+###  3.2. Technical Properties
 
 ##### id
 
-The URI that identifies this resource. It is _RECOMMENDED_ that an HTTPS URI be used for all resources. If this resource is only available embedded (see the [terminology section][prezi-api-3-terminology] for an explanation of "embedded") within another resource, such as a Canvas within a Manifest, then the URI _MAY_ be the URI of the encapsulating resource with a unique fragment on the end. This is not true for Canvases, which _MUST_ have their own URI without a fragment.
+The URI that identifies this resource. It is _RECOMMENDED_ that an HTTPS URI be used for all resources defined by this specification. If this resource is only available embedded (see the [terminology section][prezi-api-3-terminology] for an explanation of "embedded") within another resource, such as a Range within a Manifest, then the URI _MAY_ be the URI of the encapsulating resource with a unique fragment on the end. This is not true for Canvases, which _MUST_ have their own URI without a fragment.
```

Comment:
HTTPS -> HTTP(S)

Reply:
:+1: fixed. Though not introduced by this PR.